src/api/services/task_queue.py

```diff
@@ -34,7 +34,11 @@ def list_tasks_functions(self, context: Context, args) -> Tuple[list, MassEnergi
 
   def create_task(self, context, args) -> Tuple[dict, MassEnergizeAPIError]:
     if not args.get("job_name", None):
-      return None, MassEnergizeAPIError("Automatic proccess is required")
+      return None, MassEnergizeAPIError("Automatic process is required")
```

Comment:
Please make this "Automatic task name not supplied"

Reply:
ok